File: backend/api/src/main/scala/com/github/ai/split/api/response/DeleteExpenseResponse.scala
package com.github.ai.split.api.response

import com.github.ai.split.api.GroupDto
import zio.json.{DeriveJsonEncoder, JsonEncoder}

case class DeleteExpenseResponse(
  group: GroupDto
)

object DeleteExpenseResponse {
  implicit val encoder: JsonEncoder[DeleteExpenseResponse] = DeriveJsonEncoder.gen[DeleteExpenseResponse]
}

File: backend/api/src/main/java/com/github/ai/split/api/response/DeleteExpenseResponse.java
package com.github.ai.split.api.response;

import com.github.ai.split.api.GroupDto;
public class DeleteExpenseResponse {

    public GroupDto group = new GroupDto();

    public DeleteExpenseResponse() {}

    public DeleteExpenseResponse(GroupDto group) {
        this.group = group != null ? group : new GroupDto();
    }
}